doc/install.rst

```diff
@@ -128,30 +128,23 @@ want):
 
 .. tab-set::
```

Comment:
At line 120, we have:

> First, we must configure conda to get packages from the conda-forge channel 
>
>     conda config --prepend channels conda-forge

This line is not necessary if users use Mambaforge + mamba, but are required if users use conda or install mamba in other ways.

I'm unsure if we should remove these lines and always add the `--channel conda-forge` option to `conda`/`mamba` commands.

Reply:
Maybe mention that this step is not needed if `Mambaforge` is used? Or we could just leave it, since running this command under Mambaforge doesn't hurt actually.

Reply:
> Or we could just leave it, since running this command under Mambaforge doesn't hurt actually.

Sounds good to me.